### ISSUE TITLE
I'm interested in controlling timing between TCM and E_ink.

### ISSUE THREAD

**Issue by XzeroXone** [babean/Photon-E-ink#1]
The timing between TCM and E_ink is not mentioned in the DevelopersGuide,But I am more interested in it.
Signal timing on pins such as CKV \STV_IN\OE123\SO\SI\SCLK\CS.
I have some EDP Panel, and I want to design a project to drive EDP Panel directly, which will be more concise.  but I don't have a TCM. This bothers me.
Will your TCM still work? Are you considering reselling it? I want to analyze this interface timing. 
Thank you !
![E_Ink](https://user-images.githubusercontent.com/111035714/184165535-93dd6c1d-39a3-44ab-ad71-ec048b3095c9.jpg)

**Comment by babean** [comment]
This project used Pervasive's TCM and provides Particle.IO drivers to act as host.

**Comment by XzeroXone** [comment]
> This project used Pervasive's TCM and provides Particle.IO drivers to act as host.

Yes, it is more difficult to design Host alone to drive E_Ink. If I want to adjust the grayscale, I need precise timing. I did not find a technical manual on the PDI website about controlling timing. I'll keep trying to find more information.

**Comment by babean** [comment]
You will still need to talk to an eink driver as these are not standard logic levels. I started to design a board to do this with a TI chip. I'll see if I can dig it up.

**Comment by mr0bird** [comment]
Hello, can you share the official hardware sch? I bought a panel and try to make it work.
